README.md

```diff
@@ -557,6 +558,66 @@ For detailed documentation on the InstructLab LLMs and their functions, see the
    ilab model chat --model ~/.cache/instructlab/models/mixtral-8x7b-instruct-v0.1.Q4_K_M.gguf --temperature 0.5
    ```
 
+### 📇 Configure retrieval-augmented generation (developer preview)
+
+If you want to use the developer preview of retrieval-augmented generation (RAG), you need to enable developer preview features by setting `ILAB_FEATURE_SCOPE` to `DevPreviewNoUpgrade`. This tells InstructLab to enable developer preview features that might not be upgradable.  For many popular shells, this is done via the following:
+
+   ```shell
+   export ILAB_FEATURE_SCOPE=DevPreviewNoUpgrade
+   ```
+
+#### Converting documents
+
+Once you have enabled developer preview features, the first step is to convert your content into a structured form and then index that content for RAG to use with `ilab model chat`.  There are three options for conversion:
+
+*Converting documents from a directory:* If you have all the source documents (e.g., PDF, Markdown) that you want to convert in a single local directory, you can convert them:
+
+   ```shell
+      ilab rag convert --input-directory <path-to-document-directory> --output-directory <path-to-converted-document-directory> 
+   ```
+
+This capability has been tested with PDF, Markdown, and Microsoft Word format documents.  Any of the [formats supported by Docling](https://ds4sd.github.io/docling/reference/document_converter/#docling.document_converter.InputFormat) should work.
+
+*Converting documents from a taxonomy:* If you do not specify an input directory, the `convert` command will pull from your taxonomy instead.  For example, to use all of the documents in all of the knowledge nodes in your taxonomy you can run:
+
+   ```shell
+   ilab rag convert --taxonomy-base=empty --output-directory <path-to-converted-document-directory> 
+   ```
+
+*Converting documents within SDG:* The SDG commands will convert documents along with its primary goal (to generate synthetic data).  If you ran any configuration of `ilab data generate`, then the knowledge documents in the taxonomy that generated synthetic data will already be converted to a structured form.  In this case, there is no need to run the `ilab rag convert` command again.  For example, the following command mentioned in the [🚀 Generate a synthetic dataset](#-generate-a-synthetic-dataset) section converts all of the documents referenced in the entire contents of your taxonomy:
+
+   ```shell
+   ilab data convert --taxonomy-base=empty
+   ```
```

Comment:
this one not specify the `--output-directory`, seems would be better to tell the default output dir in the description.

Reply:
The point I am trying to make is that the default *input* to `ilab rag ingest` is the default *output* from `ilab data convert` so if you are following that path you don't need to specify the directories at all.  I will try to make that clear without getting into specifics of the default values which users can get from running with `--help` and that we can get into in reference documentation if/when we make that.